lib/help.js

```diff
@@ -11,7 +11,14 @@ var helpjson = {
 	'submit'			: 'To submit the code to JavaAutolab and fetch the results',
 	'help'				: 'Print help manual'
 };
+/**
+* Displays the list of autolab commands along with their functions.
+* @function help
+* @param {null}
+* @return {object} object
```

Comment:
The module help.js returns object so I was a bit confused what to write in the return statement